pkgs/by-name/rm/rmenu/package.nix

```diff
@@ -0,0 +1,83 @@
+{ fetchFromGitHub
+, glib
+, gtk3
+, lib
+, libsoup_3
+, networkmanager
+, pkg-config
+, rustPlatform
+, webkitgtk_4_1
+, wrapGAppsHook
+}:
+rustPlatform.buildRustPackage rec {
+  version = "1.2.0";
```

Comment:
pname should come before the version number. 

Please run nixpkgs-hammering (`nix run github:jtojnar/nixpkgs-hammering -- rmenu`) and also look at the other suggestions.

Reply:
ran hammering, put pname before version and inputs before postInstall, all other aspects are part of rust derivation builder.